nsqd/protocol_v2_test.go

```diff
@@ -227,11 +229,10 @@ func TestClientTimeout(t *testing.T) {
 		default:
 			_, err := nsq.ReadResponse(conn)
 			if err != nil {
-				goto done
+				break loop
```

Comment:
this seems like a stylistic change that feels inconsistent with the rest of the codebase, would also prefer dropping this.